controllers/upgrade_handlers.go

```diff
@@ -119,6 +119,15 @@ func (u *UpgHandler) PrePivot(ctx context.Context, ibu *lcav1alpha1.ImageBasedUp
 		u.resetProgressMessage(ctx, ibu)
 	}
 
+	// validate health first
```

Comment:
nit: this comment i think can be removed since the log below it says the same same thing.

Reply:
Removed